standard_logger.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+
+def login():
```

Comment:
OK, a wrapper module sounds good. This method is strange though, as I can't see how you would use it from the rest of the code.
If we want a wrapper, there should be a way to perform logging at different severities (INFO, WARNING, ERROR, etc.).

Regarding the creation of a new clean pull request, if you don't know how to do it, feel free to ask me on IRC.